py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -19,15 +19,15 @@ def amax(
     source_ir: Optional[SourceIR],
     name: str,
     input_val: TRTTensor,
-    dim: Union[int, Sequence[int]],
+    dim: Sequence[int] = [],
```

Comment:
Minor suggestion/comment - should the `amin` and `amax` converters have any default arguments? It seemed most of the `impl` implementations do not have any default arguments, and instead let the converter in `aten_ops_converters.py` dictate the defaults.

Reply:
Well, that's a good question. I also thought about it. At first I didn't think too much about it, just letting `aten_ops_converters.py` dictate default arguments, but later, I started to add default arguments that the schema provides in the function definition, because I was thinking maybe some functions need to call basic functions from `impl`. In this case, we can directly use the default arguments if needed. Do you think which one is better? adding default arguments in the function definition in `impl` or not?

Reply:
I think the default arg could be included in the `impl` as well because of the point which you mentioned. In that case don't you think we should just keep the case in which the `dim` arg is `None`?

Reply:
@apbose Thanks for the comment! I know it's kind of unintuitive but based on the [schema](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml), the defination of `amin` is:
```
- func: amin(Tensor self, int[1] dim=[], bool keepdim=False) -> Tensor
  variants: function, method
  structured_delegate: amin.out
  tags: core
```
So I think it would be better to keep consistency by using `dim=[]`